package.json

```diff
@@ -180,7 +182,8 @@
       "one-app-statics/package.json"
     ],
     "skip": {
-      "tag": true
+      "tag": true,
+      "changelog": true
```

Comment:
Skipping the changelog generation as we do not need standard version to generate this